content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -100,6 +100,13 @@ If an event or a request goes beyond any of the following limitations, it is rej
 | Maximum event size                       | 256 KB       |
 | Maximum intake payload size              | 5 MB         |
 
+## Cross origin read blocking warning
+
+When starting session replay recording with `startSessionReplayRecording()` on Chromium based browser, a CORB warning is printed in the console:
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This is not related to `startSessionReplayRecording`. The warning is shown using RUM or Logs alone. (The warning is shown because the intake V2 returns a non-empty JSON object... previously it was used mainly for Session Replay, but now it is used for all intakes)

Reply:
Okay, make sens. I'll update the doc